runtime/util/romclasswalk.c

```diff
@@ -138,6 +138,8 @@ void allSlotsInROMClassDo(J9ROMClass* romClass,
 	SLOT_CALLBACK(romClass, J9ROM_U32,  romClass, memberAccessFlags);
 	SLOT_CALLBACK(romClass, J9ROM_U32,  romClass, innerClassCount);
 	SLOT_CALLBACK(romClass, J9ROM_SRP,  romClass, innerClasses);
+	SLOT_CALLBACK(romClass, J9ROM_U32,  romClass, enclosedInnerClassCount);
+	SLOT_CALLBACK(romClass, J9ROM_SRP,  romClass, enclosedInnerClasses);
```

Comment:
Is an equivalent update required for `jdmpview`?

Reply:
There is no need to do so given the code related to `enclosedInnerClasses` (filtered from the `InnerClass` attribute) is only used to deal with a special case when the enclosing class is not the direct outer class of the enclosed inner classes  to match the RI's behaviour, which is not explicitly specified in the VM Spec.